gpu-operator/release-notes.rst

```diff
@@ -31,8 +31,96 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
    GPU Operator beta releases are documented on `GitHub <https://github.com/NVIDIA/gpu-operator/releases>`_. NVIDIA AI Enterprise builds are not posted on GitHub.
 
+
 ----
 
+.. _v24.3.0:
+
+24.3.0
+======
+
+.. _v24.3.0-new-features:
+
+New Features
+------------
+
+
+* Added support to enable NVIDIA GDRCopy v2.4.1.
+
+  When you enable support for GDRCopy, the Operator configures the GPUCopy Driver container image
+  as a sidecar container in the GPU driver pod.
+  The sidecar container compiles and installs the gdrdrv Linux kernel module.
+  This feature is supported on Ubuntu 22.04 and RHCOS operating systems and on X86_64 and ARM64 architectures.
+
+  Refer to :ref:`Chart Customization Options` for more information about the ``driver.gdrcopy`` field.
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.54.15.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit version v1.15.0
+    - NVIDIA MIG Manager version v0.6.0
```

Comment:
https://gitlab.com/nvidia/kubernetes/gpu-operator/-/merge_requests/1071
```suggestion
    - NVIDIA MIG Manager version v0.7.0
```

Reply:
I still see v0.6.0 listed.

Reply:
Must be human error, sorry.  I understood your comment and I even know where to look for the single version of the truth.  TY.